jobclient/src/main/java/com/twosigma/cook/jobclient/Job.java

```diff
@@ -371,6 +413,7 @@ private Job(UUID uuid, String name, String command, Double memory, Double cpus,
         } else {
             _container = null;
         }
+        _labels = ImmutableMap.copyOf(labels);
```

Comment:
similarly, we won't be able to initialize with a label map and then add additional labels